example/server.js

```diff
@@ -21,10 +22,12 @@ import {
   GraphQLString,
   GraphQLID,
   GraphQLList,
-} from 'graphql';
+} = require('graphql');
 
 const app = express();
+
 app.use(express.static(__dirname));
```

Comment:
Adding the root of the project as another static directory for css and other compiled assets (graphiql.js)